src/components/gift_themed_list_page/Banner.tsx

```diff
@@ -0,0 +1,74 @@
+import publicApi from '@/apiClient/publicApi';
+import styled from '@emotion/styled';
+import axios from 'axios';
+import { useEffect, useState } from 'react';
+import { useNavigate, useParams } from 'react-router-dom';
+
+const Container = styled.div<{ backgroundColor: string }>`
+  display: flex;
+  flex-direction: column;
+  justify-content: flex-start;
+  align-items: flex-start;
+  width: 100%;
+  height: auto;
+  margin-top: 2.8rem;
+  box-sizing: border-box;
+  background-color: ${({ backgroundColor }) => backgroundColor};
+  padding-top: ${({ theme }) => theme.spacing.spacing6};
+  padding-bottom: ${({ theme }) => theme.spacing.spacing5};
+  padding-left: ${({ theme }) => theme.spacing.spacing4};
+`;
+
+const ThemeName = styled.div`
+  ${({ theme }) => theme.typography.label1Bold}
+  color: white;
+`;
+
+const Title = styled.div`
+  ${({ theme }) => theme.typography.title1Bold}
+  margin-top: ${({ theme }) => theme.spacing.spacing2};
+  color: white;
+`;
+
+const Description = styled.div`
+  ${({ theme }) => theme.typography.title2Regular}
+  margin-top: ${({ theme }) => theme.spacing.spacing1};
+  color: white;
+`;
+
+export const Banner = () => {
+  const navigate = useNavigate();
+  const { id } = useParams();
+  const [themeName, setThemeName] = useState('');
+  const [title, setTitle] = useState('');
+  const [description, setDescription] = useState('');
+  const [backgroundColor, setBackgroundColor] = useState('');
+
+  useEffect(() => {
+    const getData = async () => {
+      try {
+        const response = await publicApi.get(`/api/themes/${id}/info`);
+        setThemeName(response.data.data.name);
+        setTitle(response.data.data.title);
+        setDescription(response.data.data.description);
+        setBackgroundColor(response.data.data.backgroundColor);
+        console.log(response.data.data);
```

Comment:
구조분해할당을 통해서 중복되는 부분을 줄일 수 있겠네용!

```suggestion
        const response = await publicApi.get(`/api/themes/${id}/info`);
        const { data } = response.data
        const { name, title, description, backgroundColor } = data
        setThemeName(name);
        setTitle(title);
        setDescription(description);
        setBackgroundColor(backgroundColor);
        console.log(response.data.data);
```